src/components/starbucks-input/StarbucksInput.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+function StarbucksInput({name, value, onChange, error}) {
+  return (
+    <>
+    <input 
+    id="sbInput" 
+    type="textbox" 
+    name={name} 
+    value={value} 
+    onChange={(e) => onChange(e.target.value)}
```

Comment:
A few pieces of feedback:
1. add onBlur and onFocus event
2. add data-test-id
3. can we also allow ...rest ?